packages/babel-plugin-transform-function-sent/test/fixtures/generator-kinds/class-method/expected.js

```diff
@@ -0,0 +1,12 @@
+function _skipFirstGeneratorNext(fn) { return function () { var it = fn.apply(this, arguments); it.next(); return it; }; }
+
+class Foo {
+  gen() {
+    return _skipFirstGeneratorNext(function* () {
```

Comment:
I'm not sure I follow, but it's not really important for this PR. We can do it in a follow up.